docs/Upgrade-Guide/upgrade_to_4.1.md

```diff
@@ -42,18 +43,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
     #gluster volume status
 ```
 
-7. Self-heal all gluster volumes by running
+7. Start glustereventsd service (if it was enabled before), either using the command below, or through other means,
+```sh
+    #systemctl start glustereventsd
+```
+
+8. Self-heal all gluster volumes by running
 ```sh
     #for i in `gluster volume list`; do gluster volume heal $i; done
 ```
 
-8. Ensure that there is no heal backlog by running the below command for all volumes
+9. Ensure that there is no heal backlog by running the below command for all volumes
 ```sh
     #gluster volume heal <volname> info
 ```
-> NOTE: If there is a heal backlog, wait till the backlog is empty, or the backlog does not have any entries needing a sync to the just upgraded server, before proceeding to upgrade the next server in the pool
+> **NOTE:** If there is a heal backlog, wait till the backlog is empty, or the backlog does not have any entries needing a sync to the just upgraded server, before proceeding to upgrade the next server in the pool
```

Comment:
Before proceeding to upgrade the next server in the pool it is recommended to check the heal backlog. If there is a heal backlog, it is recommended to wait until the backlog is empty, or, the backlog does not contain any entries requiring a sync to the just upgraded server.

Reply:
Done